engines/bastion/app/assets/javascripts/bastion/components/bst-modal.directive.js

```diff
@@ -59,6 +59,7 @@ angular.module('Bastion.components').directive('bstModal',
 
                     modalInstance.result.then(function () {
                         scope.action();
+                    }, function() {
```

Comment:
Why'd you need to add this in?

Reply:
Uggh.. 😬  Ya, the bastion library somehow was suppressing a weird console warning on modal close. When I overrode the modal footer for repo deletion to add the customization with checkbox and button disabling, it started appearing. 
This is to suppress that error back. More details here: https://github.com/angular-ui/bootstrap/issues/6501#issuecomment-292881004

Reply:
Oh weird! Okay, makes sense then.